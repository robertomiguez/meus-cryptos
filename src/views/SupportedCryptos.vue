<template>
  <div class="wrapper">
    <listCrypto/>
  </div>
</template>

<script>
import listCrypto from '../components/ListCrypto.vue'

export default {
  name: 'SupportedCryptos',
  components: {
    listCrypto
  }
}
</script>

<style scoped>
.wrapper {
  padding: 68px 0px 30px 0px;
}
</style>
