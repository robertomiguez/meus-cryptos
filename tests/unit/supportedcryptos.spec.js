import { shallowMount } from '@vue/test-utils'
import SupportedCryptos from '@/views/SupportedCryptos'

describe('SupportedCryptos', () => {
  const build = () => {
    const wrapper = shallowMount(SupportedCryptos)

    return {
      wrapper
    }
  }

  it('renders the component', () => {
    // arrange
    const { wrapper } = build()
    // assert
    expect(wrapper.html()).toMatchSnapshot()
  })
})