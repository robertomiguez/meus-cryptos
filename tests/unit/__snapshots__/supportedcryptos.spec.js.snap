// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SupportedCryptos renders the component 1`] = `
<div class="wrapper">
  <listcrypto-stub></listcrypto-stub>
</div>
`;
